docs/core/additional-tools/xmlserializergenerator-instructions.md

```diff
@@ -0,0 +1,45 @@
+ # Using Microsoft Xml Serializer Generator on .NET Core
+
+Like the Xml Serializer Generator (sgen.exe) on desktop, Microsoft.XmlSerializer.Generator NuGet package is the solution for .NET Core and .NET Standard Libraries. It creates an Xml serialization assembly for types contained in an assembly to improve the startup performance of Xml serialization when serializing or de-serializing objects of those types using XmlSerializer. 
+
+You can start using the tool today following the instructions below. 
+
+## Prerequisites
+
+The following is required for Microsoft.XmlSerializer.Generator to work. You can use command `dotnet --info` to check which versions of .NET Core SDK and runtime you may already have installed.
+
+* [.NET Core SDK 2.0.2 or later](https://www.microsoft.com/net/download/windows)
+* [.NET Core runtime 2.0.3 or later](https://github.com/dotnet/core/blob/master/release-notes/download-archives/2.0.3.md)
+  
+## Instructions
+
+Here are the step by step instructions on how to use Xml Serializer Generator in a .NET Core console application.
+
+1. Create a .NET Core console application, e.g. create an app named 'MyApp' with the command,
+    ```
+    dotnet new console --name MyApp
+    ```
+1. Edit the .csproj and add a reference to the Microsoft.XmlSerializer.Generator package. For example,
```

Comment:
choose one way of doing it. since you're already on CLI, continue to use the CLI.

You can then tell them they can open the project file and see that the following lines were added.

Something like:

1. Add a reference to the Microsoft.XmlSerializer.Generator package:

`dotnet add package Microsoft.XmlSerializer.Generator -v 1.0.0-preview1-25915-02`

After running the command, the following lines are added to your project file:
 ```xml
    <ItemGroup>
      <DotNetCliToolReference Include="Microsoft.XmlSerializer.Generator" Version="1.0.0-preview1-25915-02" />
    </ItemGroup>
```